frontend/packages/topology/integration-tests/support/page-objects/topology-add-options-po.ts

```diff
@@ -0,0 +1,13 @@
+export const topologyAddOptionsPO = {
+  samples: '[data-test-action="Samples"]',
+  importFromGit: '[data-test-action="Import from Git"]',
+  containerImage: '[data-test-action="Container Image"]',
+  catalog: '[data-test-action="From Catalog"]',
+  database: '[data-test-action="Database"]',
+  operatorBacked: '[data-test-action="Operator Backed"]',
+  helmCharts: '[data-test-action="Helm Charts"]',
+  eventsource: '[data-test-action="Event Source"]',
+  channel: '[data-test-action="Channel"]',
```

Comment:
Make a dynamic function for it and use it

Reply:
Like this [data-test-action="${actionName}"]

Reply:
updated